extensions/amp-call-tracking/amp-call-tracking.md

```diff
@@ -32,7 +32,7 @@ limitations under the License.
   </tr>
   <tr>
     <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
-    <td>fill, fixed, fixed-height, flex-item, responsive</td>
+    <td>container, fill, fixed, fixed-height, flex-item, nodisplay, responsive</td>
```

Comment:
The validator proto and the JS implementation are inconsistent. That was a mistake on my part.  Created #7732 to fix it.